cfn-resources/cloud-backup-schedule/test/cloud-backup-schedule.sample-cfn-request.json

```diff
@@ -1,51 +1,29 @@
 {
   "desiredResourceState": {
     "ApiKeys": {
-      "PublicKey": "",
-      "PrivateKey": ""
+      "PublicKey": "xfqavjxw",
```

Comment:
watch this keys @govardhanpagidi, if they are active keys lets remove them

Reply:
removed